pkg/tagger/collectors/docker_extract.go

```diff
@@ -76,17 +76,17 @@ func dockerExtractLabels(tags *utils.TagList, containerLabels map[string]string,
 		switch labelName {
 		// Docker swarm
 		case "com.docker.swarm.service.name":
-			tags.AddLow("swarm_service", labelValue)
```

Comment:
I feel like all these tags (swarm_service, rancher_stack, nomad_job.....) should stay as low cardinality, as they don't churn if you re-create your pod. WDYT?
`mesos_task` can move from high to orch, that's true